src/main.js

```diff
@@ -147,10 +148,17 @@ function renderOverlay() {
     el.innerHTML = `<header style='position:fixed;top:0;left:0;right:0;display:flex;align-items:center;justify-content:space-between;padding:1rem;height:2.5rem;background-color:#333;z-index:1;'>\
         <div style='font-family:monospace;font-size:1.5em;cursor:pointer;' onclick='location.reload()'>InstagramUnfollowers</div>\
         <label style='display:flex;cursor:pointer;'><input type='checkbox' class='iu_include-verified-checkbox' />&nbsp;Include verified</label>\
+        <div class="progress-bar" style="display:none; width: 120px;height: 30px;border-radius: 5px;margin: 20px 10px;border: 1px solid #7b7777;overflow: hidden;position: relative;">
+            <span class="loading-percentage" style="width: 0;height: 100%;display: block;color: white;line-height: 30px;position: absolute;text-align: end;padding-right: 5px;background-color: #7b7777;">0%</span>
+        </div>
```

Comment:
Right now it looks a bit weird since the percentage text jumps from left to right as it's climbing.

So here I think we can benefit from having one span for the progress background,
and another one for the percentage.

That way, we can play around with the style to make the percentage span always in the middle.